src/site/content/en/secure/cross-origin-isolation-guide/index.md

```diff
@@ -18,11 +18,79 @@ tags:
 ---
 
 This guide shows you how to enable cross-origin isolation. Cross-origin
-isolation is required if you want to use SharedArrayBuffer,
+isolation is required if you want to use
+[SharedArrayBuffer](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer),
 [`performance.measureUserAgentSpecificMemory()`](/monitor-total-page-memory-usage/),
 or the JS Self-Profiling API.
 
-## Do an impact analysis
+{% Aside 'objective' %}
+If you landed this page by a SharedArrayBuffer deprecation message, it's likely
+either your website or one of the resources you emebed to your website is using
+SharedArrayBuffer. When Chrome hits 91, the functionality that has been
+available through SharedArrayBuffer will no longer work without cross-origin
+isolation. Take one of the following actions to fix the issue in advance.
+
+* Turn on the cross-origin isolation to keep using SharedArrayBuffer.
+* Notify the third-party vendor/provider that actually uses SharedArrayBuffer to
+  take action.
+{% endAside %}
+
+## Determine where in your website a SharedArrayBuffer is used
+
+If you are not sure where in your site a SharedArrayBuffer is used, there are
+two ways:
+
+* [Basic] Using Chrome DevTools
+* [Advanced] Using Deprecation Reporting
+
+Skip to [Do an impact analysis](:analysis) if you already know where you are
+using SharedArrayBuffer.
+
+### [Basic] Using Chrome DevTools
+
+Chrome has a tool for developers to inspect the website. You can identify the
+issue using it.
+
+1. [Open the Chrome
+   DevTools](https://developers.google.com/web/tools/chrome-devtools/open) on
+   the page you suspect it might be using a SharedArrayBuffer.
+2. Select the **Console** panel.
+3. The following message will show up if SharedArrayBuffer is used:
+      ```text
+      [Deprecation] SharedArrayBuffer will require cross-origin isolation as of M91, around May 2021. See https://developer.chrome.com/blog/enabling-shared-array-buffer/ for more details. common-bundle.js:535
+      ```
+4. The filename and the line number indicates where the SharedArrayBuffer comes
+   from. If it's a third-party library, please contact the developer to fix the
+   issue. If it's implemented as part of your website, follow the guide below to
+   enable cross-origin isolation.
```

Comment:
```suggestion
### Using Chrome DevTools

[Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/open)
allows developers to inspect websites. 

1. [Open the Chrome
   DevTools](https://developers.google.com/web/tools/chrome-devtools/open) on
   the page you suspect might be using `SharedArrayBuffer`.
2. Select the **Console** panel.
3. If the page is using `SharedArrayBuffer`, the following message will show up:
      ```text
      [Deprecation] SharedArrayBuffer will require cross-origin isolation as of M91, around May 2021. See https://developer.chrome.com/blog/enabling-shared-array-buffer/ for more details. common-bundle.js:535
      ```
4. The filename and the line number at the end of the message (for example, `common-bundle.js:535`) 
   indicate where the `SharedArrayBuffer` is coming from. If it's a third-party library, 
   contact the developer to fix the issue. If it's implemented as part of your website, follow 
   the guide below to enable cross-origin isolation.
```